utils/frame/remote-externalities/Cargo.toml

```diff
@@ -13,16 +13,15 @@ readme = "README.md"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-jsonrpc-core-client = { version = "15.1.0", features = ["http"] }
-sc-rpc-api = { version = "0.9.0", path = "../../../client/rpc-api" }
-sc-rpc = { version = "3.0.0", path = "../../../client/rpc" }
-futures = "0.3"
+jsonrpsee-http-client = { version = "0.2.0-alpha", default-features = false, features = ["tokio02"] }
+# Needed by `jsonrpsee-proc-macros`
```

Comment:
I hope there exists an upstream issue to fix this. Having macros depend on other crates isn't such a nice experience.

Reply:
https://github.com/paritytech/jsonrpsee/issues/214